src/auxiliary/special_elixirs.jl

```diff
@@ -36,6 +36,16 @@ julia> redirect_stdout(devnull) do
 ```
 """
 function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
+    # Check that all kwargs exist as assignments
+    code = read(elixir, String)
+    expr = Meta.parse("begin \n$code \nend")
+    expr = insert_maxiters(expr)
+
+    for (key, val) in kwargs
+        # This will throw an error when `key` is not found
+        find_assignment(expr, key)
+    end
```

Comment:
Did you consider checking the existence of the keywords to replace in `replace_assignments` below?

Reply:
I did consider it, but `replace_assignments` is not called for the full expression, and instead passed as a function to `Base.include`:
```julia
Base.include(ex -> replace_assignments(insert_maxiters(ex); kwargs...), mod, elixir)
```
I couldn't come up with a nice solution to track the found assignments.